charts/hivemq-operator/operator-tmpls/cluster-stateful-set.yaml

```diff
@@ -16,8 +16,14 @@ metadata:
     name: "{{ spec.name }}"
     uid: "{{ spec.metadata.uid }}"
 spec:
-  serviceName: hivemq-{{ spec.name }}-cluster
   replicas: {{ spec.nodeCount }}
+  serviceName: hivemq-{{ spec.name }}-cluster
+  podManagementPolicy: Parallel
+  updateStrategy:
+    type: OnDelete
+  # TODO get this from the CR
```

Comment:
TODO still valid ?

Reply:
The ToDo is also still in the upstream operator template changes - needs to be removed there when we sync the final template manifestes with the helm chart